.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
- [ ] Kindly setup linters for javascript. I can see your linters are checking your html and css files but kindly add your re-configure your linters to check javascript files using this [code of linters.yml](https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.github/workflows/linters.yml) file 😊